scss/_product-page.scss

```diff
@@ -75,6 +75,9 @@ $decalTop:48px;
         }
         .attr_text {
             padding-left:0.5rem;
+            .attribute_missing::before {
```

Comment:
The attribute missing is not really a warning, it's more an explanation of why an attribute is not computed (and thus shown in grey), so we should not display a warning sign there.

Reply:
So hall I remove all the changes to the attribute panel?

Reply:
Yes, I think the attribute panel should stay as it was.